sdk/storage/azure-storage-blob-changefeed/pom.xml

```diff
@@ -72,7 +72,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.26.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.26.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
```

Comment:
Do we need to also update the Changelog for ChangeFeed?

Reply:
Also, I also ask this, change feed is staying in beta, correct?

I think I always ask this question for this package.